spec/ddtrace/tracer_spec.rb

```diff
@@ -253,12 +327,39 @@
       end
 
       context 'when nesting spans' do
-        it 'only the top most span has a runtime ID tag' do
+        it do
+          tracer.trace('parent', service: 'service-parent') do
+            tracer.trace('child1') { |s| s.set_tag('tag', 'tag_1') }
+            tracer.trace('child2', service: 'service-child2') { |s| s.set_tag('tag', 'tag_2') }
+          end
+
+          expect(spans).to have(3).items
+
+          child1, child2, parent = spans # Spans are sorted alphabetically by operation name
```

Comment:
Minor: Have you considered encoding this note on the helper name? E.g. `spans_sorted_by_operation_name`?

Reply:
`sorted_spans` maybe, given `spans` happens about 1 billion times in our test suite :)